docs/labs/deviceplugin-kmm.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   devicePlugin:
     container:
-      image: "quay.io/myrepo/oc-dummy-device-plugin:0.1"
+      image: "quay.io/ebelarte/oc-dummy-device-plugin:0.1"
```

Comment:
Can we continue using `myrepo` or even better `<org>`? Just to make sure it is clear to the user it needs to inject some input into the yaml.